geometry_msgs/Pose pose
---
bool point_reached